README.zh-CN.md

```diff
@@ -61,23 +61,198 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
 - [EventMesh-catalog](https://github.com/apache/eventmesh-catalog): 使用 AsyncAPI 进行事件模式管理的目录服务。
 - [EventMesh-go](https://github.com/apache/eventmesh-go): EventMesh 运行时的 Go 语言实现。
 
-## 快速开始
+## 快速入门   
 
-指南:
+本节指南将指导您分别从[本地](#在本地运行-eventmesh-runtime)、[docker](#在-docker-中运行-eventmesh-runtime-)、[k8s](#在-kubernetes-中运行-eventmesh-runtime-)部署EventMesh的步骤:
+
+本节指南只是帮助您快速入门EventMesh部署，按照默认配置启动EventMesh，如果您需要更加详细的EventMesh部署步骤，请访问[EventMesh官方文档](https://eventmesh.apache.org/docs/next/introduction)。
+
+### 部署 EventMesh Store  
+
+> EventMesh现在支持`standalone`、`RocketMQ`、`Kafka`等中间件作为存储   
+> 如果是在非`standalone`模式下，需要先部署所需的`store`，以`rocketmq`模式为例: 部署[RocketMQ](https://rocketmq.apache.org/docs/quickStart/01quickstart/)
 
-[第1步: 部署EventMesh Store](https://eventmesh.apache.org/docs/instruction/store)
+### 在本地运行 EventMesh Runtime
 
-[第2步: 启动EventMesh Runtime](https://eventmesh.apache.org/docs/instruction/runtime)
+请在开始之前检查JKD版本，需要下载Java 8.
+```
+$ java -version
+java version "1.8.0_311"
+```
 
-[第3步: 运行示例](https://eventmesh.apache.org/docs/instruction/demo)
+#### 1.下载
 
-另外，如果您更喜欢使用Docker，则我们还为您提供了Docker版本的指南：
+在[EventMesh download](https://eventmesh.apache.org/download/)页面选择所需要版本的Binary Distribution进行下载,您将获得`apache-eventmesh-1.10.0-bin.tar.gz`。
+```
+tar -xvzf apache-eventmesh-1.10.0-bin.tar.gz
+cd apache-eventmesh-1.10.0
+```
 
-[第1步: 使用Docker部署EventMesh Store](https://eventmesh.apache.org/docs/instruction/store-with-docker)
+#### 2. 运行  
 
-[第2步: 使用Docker启动EventMesh Runtime](https://eventmesh.apache.org/docs/instruction/runtime-with-docker)
+编辑`eventmesh.properties`以更改EventMesh Runtime的配置（如 TCP 端口、客户端黑名单）。
+```
+vim conf/eventmesh.properties
+```
 
-[第3步: 运行示例](https://eventmesh.apache.org/docs/instruction/demo)
+指定事件存储为 RocketMQ(默认为standalone)：
+```
+# storage plugin
+eventMesh.storage.plugin.type=rocketmq
+```
+
+执行`start.sh`脚本启动EventMesh Runtime服务器。
+```
+bash bin/start.sh
+```
+如果看到`EventMeshTCPServer[port=10000] started....`, 则说明设置成功。
```

Comment:
Please follow markdown writting spefication and add a blank line between paragraphs.

`EventMeshTCPServer[port=10000] started....` will not be outputed from `start.sh`.

`EventMeshTCPServer[port=10000] started....` doesn't represent server fully up, `server state:RUNNING` does, because:

- Any of TCP,HTTP,GRPC server can be excluded from config, and the corresponding server won't start.
- Searching for `started` in log file will have too much results and searching for `RUNNING` will have a unique result.